CDLUC3/ezid
# Remove Google Analytics code/config, modify special tracking to use matomo

Closes #539 .

- Removes Google analytics configuration and tracking from the codebase.
- There were a number of special Javascript classes and functions that were set up for Google analytics, which I changed to get working with Matomo.

How to verify that tracking is working:

Either test from a server, or you can use tracking from our local machine if you use the development configuration.  Those URLs will show up as localhost:8000 in Matomo. The Matomo address is https://matomo.cdlib.org/index.php .  Choose the ezid-dev.cdlib.org site.  You should see new user visits under the "visits in real time" section.  You can see additional pages visited for each user by hovering over the icons.  I believe there is also a more detailed view.

Items with special tracking that are enabled with some special javascripts and still should show up in matomo (may need to refresh matomo page after clicks to see updates there).

- Search queries show the keywords entered for search.
- Help popovers are tracked (for example, the ? next to ARK/DOI for new identifier or other places in UI).
- Login to site is tracked.
- Contact form submitted
- Download help documents. like the page and http://ezid-dev.cdlib.org/learn/#04 and clicking to download something like the service guidelines PDF that isn't an html page.

These were the special cases I saw tracked in the extra code.  Of course it also tracks normal html pages elsewhere on the site, also.
